service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -164,23 +165,9 @@ public void notifyApplicant(final AdditionalApplicationsUploadedEvent event) {
 
         final Set<String> recipients = new HashSet<>();
 
-        if (applicant.getType() == LOCAL_AUTHORITY) {
-
-            final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
-                .caseData(caseData)
-                .secondaryLocalAuthorityExcluded(true)
-                .build();
-
-            recipients.addAll(localAuthorityRecipients.getRecipients(recipientsRequest));
-
-        } else if (applicant.getType() == SECONDARY_LOCAL_AUTHORITY) {
-
-            final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
-                .caseData(caseData)
-                .designatedLocalAuthorityExcluded(true)
-                .build();
-
-            recipients.addAll(localAuthorityRecipients.getRecipients(recipientsRequest));
+        if (applicant.getType() == LOCAL_AUTHORITY || applicant.getType() == SECONDARY_LOCAL_AUTHORITY) {
+            // do nothing
```

Comment:
Guess you could get rid of the if/else and do something like if(applicatn.getType() != LOCAL_AUTHORITY && applicant.getType() != SECONDARY_LOCAL_AUTHORITY)?

Or rather than two event listeners, you could perhaps convert notifyAllLocalAuthorities to a helper method and always call it in this method so LAs are always notified?